client/app/account/user/agreements/user-agreements.service.js

```diff
@@ -1,19 +1,84 @@
 angular.module("UserAccount").service("UserAccount.services.agreements", [
     "$http",
     "$q",
+    "$translate",
     "UserAccount.constants",
     "$cacheFactory",
-    function ($http, $q, constants, cache) {
+    function ($http, $q, $translate, constants, cache) {
         "use strict";
 
         const userAgreementsCache = cache("USER_AGREEMENTS");
 
         const proxyPath = `${constants.swsProxyRootPath}me`;
 
+        const GDPR_AGREEMENTS_INFOS = [
+            {
+                agreementId: 1800,
+                subsidiary: "DE",
+                url: "https://www.ovh.de/privacy"
+            }, {
+                agreementId: 1801,
+                subsidiary: "LT",
+                url: "https://www.ovh.lt/privacy"
+            }, {
+                agreementId: 1802,
+                subsidiary: "CZ",
+                url: "https://www.ovh.cz/privacy"
+            }, {
+                agreementId: 1803,
+                subsidiary: "IT",
+                url: "https://www.ovh.it/privacy"
+            }, {
+                agreementId: 1804,
+                subsidiary: "PT",
+                url: "https://www.ovh.pt/privacy"
+            }, {
+                agreementId: 1805,
+                subsidiary: "ES",
+                url: "https://www.ovh.es/privacy"
+            }, {
+                agreementId: 1806,
+                subsidiary: "NL",
+                url: "https://www.ovh.nl/privacy"
+            }, {
+                agreementId: 1807,
+                subsidiary: "FI",
+                url: "https://www.ovh-hosting.fi/privacy"
+            }, {
+                agreementId: 1808,
+                subsidiary: "PL",
+                url: "https://www.ovh.pl/privacy"
+            }, {
+                agreementId: 1821,
+                subsidiary: "EN",
+                url: "https://www.ovh.co.uk/privacy"
+            }, {
+                agreementId: 1822,
+                subsidiary: "IE",
+                url: "https://www.ovh.ie/privacy"
+            }, {
+                agreementId: 1826,
+                subsidiary: "FR",
+                url: "https://www.ovh.com/fr/protection-donnees-personnelles"
+            }
+        ];
+
         function getSuccessDataOrReject (response) {
             return response.status < 300 ? response.data : $q.reject(response.data);
         }
 
+        function formatList (response) {
+            if (response.data.list && response.data.list.results && response.data.list.results.length) {
```

Comment:
`_(response).get("list.results.length", 0) !== 0`

Reply:
Even if this would be a little bit more clear, I'm trying to code as much as possible in pure ES6

Reply:
```js
function baseGet(object, path) {
  path = castPath(path, object)

  let index = 0
  const length = path.length

  while (object != null && index < length) {
    object = object[toKey(path[index++])]
  }
  return (index && index == length) ? object : undefined
}
```

Looks pretty much ES6 compliant to me.

Reply:
I mean without using a library, lodash is fantastic but I donc think using it every time we can is in our interest